project/db/migrate/20221226034209_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string      :title, null: false, index: true
+      t.string      :detail, null: false
```

Comment:
@daiphone0228 
[must]

```suggestion
      t.text      :detail, null: false
```

Reply:
すみません、こちら漏れてましたね…対応しました。